applications/tasks/src/modules/kt_numbers.erl

```diff
@@ -259,6 +330,30 @@ account_id(_) -> 'false'.
 carrier_module(Data) ->
     lists:member(Data, knm_carriers:all_modules()).
 
+-spec ported_in(ne_binary()) -> boolean().
+ported_in(Cell) -> is_cell_boolean(Cell).
+
+-spec 'cnam.inbound'(ne_binary()) -> boolean().
+'cnam.inbound'(Cell) -> is_cell_boolean(Cell).
+
+-spec 'prepend.enabled'(ne_binary()) -> boolean().
```

Comment:
For someone that implemented a task, you should know why this is named that way.
BTW tickies *are* needed here and this is the only justifiable usage of tickies possible: when removing tickies induces a parse error.